docs/DeveloperGuide.md

```diff
@@ -80,31 +80,56 @@ The following sequence diagram shows how the Priority operation works:
   * Pros: Will use less memory (No need to show the parser field).
   * Cons: Increased coupling
 
-### \[Proposed\] Policy feature
+### Policy feature
 
-#### Proposed Implementation
+#### Implementation
 
-Policy (class) is a field in Person that is uniquely different from the current fields in Person
- such as phone, address, email, etc. A Person can have up to a single Policy. 
- 
- Policy class contains 3 attributes: String name, String description.
+Policy class is implemented as shown in this class diagram:
 
-Prior to adding a Policy field to a Person, User creates the Policy objects 
-via `addp` (Add Policy Command). A collection stores these Policy objects to be referenced and
- a json file stores Policy objects that are created. 
- 
- A user can then add one of these Policy objects as a field in a Person object by specifying with 
-  `z/ [POLICY_NAME]` during Add Command.
-  
-Sequence diagram to create new Policy:
+![Policy0](images/PolicyClassDiagram.png)
+
+PolicyName and PolicyDescription are separate classes rather than String fields.
+Implenting Policy class this way conforms to the same structure as Person class where String fields are
+their own classes.
+It also allows for abstraction of methods specified for each of the field classes such as 
+checking for validity of their String field.
```

Comment:
Perhaps we can say ...checking for validity of each of the individual object's input

Reply:
hmm... you're right sounds better